src/lib/walletStation.js

```diff
@@ -15,133 +13,224 @@ export const broadcastModeObj = {
     BROADCAST_MODE_UNSPECIFIED: "BROADCAST_MODE_UNSPECIFIED",
 };
 
+export const gasDefault = { gas: "200000", amount: '0' }
```

Comment:
We don't need this because with the new cosmwasm version, there's an option 'auto' for the `fee` parameter of client.signAndBroacast function. like: client.signAndBroadcast(address, messages, 'auto'). The function will simulate the tx & estimate the gas for us

Reply:
Thanks! let me double check and change them